### DISCUSSION TITLE
My domain is redirecting to some weird/spammy URL?

### DISCUSSION THREAD

**Discussion by rishabhsdev** [EasyEngine/easyengine#1648] [Support Requests]
I have a WordPress website hosted hosted on Ubuntu VPS using easyengine (www.foolishtaan.com), but the non-www version of my website(foolishtaan.com) is redirecting to some weird/spammy URL.

I tried checking the .htaccess file for rewrite rules, but it does not exist.

So what else could be causing this redirection? What are some other files that can cause this issue?

I have 2 other websites hosted on the same VPS using easyengine (both websites are currently disabled), their domain also redirects to the same spammy URL.

**Comment by MichaelA2014** [comment]
Probably some JS redirect hidden in a nulled theme or plugin you used...

**Comment by craftogrammer** [comment]
This is not related to EasyEngine bro. If you can't deal with them then I recommend you to don't do selfhosting. Try to understand things.
